crates/biome_js_analyze/src/lint/nursery/use_consistent_member_accessibility.rs

```diff
@@ -225,7 +225,8 @@ declare_lint_rule! {
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct ConsistentMemberAccessibilityOptions {
-    pub accessibility: Accessibility,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub accessibility: Option<Accessibility>,
```

Comment:
This is the only rule that required a `Option` because `serde` didn't understand `default` for an enum, not sure why. Maybe a bug? I didn't investigate much, though. The emitted schema is correct now

Reply:
`#[serde(default)]` only works for types that have the `Default` trait implemented. For enums, you can derive it, but you must specify which variant is the default one using the `#[default]` annotation.

Reply:
This is already the case; that's why I find it weird. 🤔

Reply:
Ok, the code should use `serde(default)` now. I was experiencing some weird issues, locally. Run a `cargo clean` and now it works.